webapps/frontend/app/viewmodels/site/vdisk-detail.js

```diff
@@ -162,6 +163,12 @@ define([
             self.vDisk().loadParentConfiguration();
             self.loadStorageRouters();
         };
+        self.removeSnapshot = function(snapshotid) {
+            api.post('vdisks/' + self.vDisk().guid() + '/removesnapshot', {
```

Comment:
I'd like to have a confirmation dialog here. There are a bunch of these around the code so you should have something to base on